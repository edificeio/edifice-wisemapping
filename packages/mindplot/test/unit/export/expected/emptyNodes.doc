<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Word.Document"?><w:wordDocument xmlns:sl="http://schemas.microsoft.com/schemaLibrary/2003/core" xmlns:w="http://schemas.microsoft.com/office/word/2003/wordml" xmlns:w10="urn:schemas-microsoft-com:office:word" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882" xmlns:wx="http://schemas.microsoft.com/office/word/2003/auxHint" xmlns:aml="http://schemas.microsoft.com/aml/2001/core" xmlns:o="urn:schemas-microsoft-com:office:office">
<o:DocumentProperties>
<o:Title/>
</o:DocumentProperties>
<w:styles>
<w:versionOfBuiltInStylenames w:val="4"/>
<w:latentStyles w:defLockedState="off" w:latentStyleCount="156"/>
<w:style w:type="paragraph" w:default="on" w:styleId="Normal">
<w:name w:val="Normal"/>
<w:rsid w:val="00831C9D"/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:sz w:val="24"/>
<w:sz-cs w:val="24"/>
<w:lang w:val="EN-US" w:fareast="EN-US" w:bidi="AR-SA"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading1">
<w:name w:val="heading 1"/>
<wx:uiName wx:val="Heading 1"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading1"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="32"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading2">
<w:name w:val="heading 2"/>
<wx:uiName wx:val="Heading 2"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading2"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="1"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading3">
<w:name w:val="heading 3"/>
<wx:uiName wx:val="Heading 3"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading3"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="2"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading4">
<w:name w:val="heading 4"/>
<wx:uiName wx:val="Heading 4"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading4"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="3"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading5">
<w:name w:val="heading 5"/>
<wx:uiName wx:val="Heading 5"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading5"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="4"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading6">
<w:name w:val="heading 6"/>
<wx:uiName wx:val="Heading 6"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading6"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="5"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading7">
<w:name w:val="heading 7"/>
<wx:uiName wx:val="Heading 7"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading7"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="6"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading8">
<w:name w:val="heading 8"/>
<wx:uiName wx:val="Heading 8"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading8"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="7"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:i/>
<w:i-cs/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading9">
<w:name w:val="heading 9"/>
<wx:uiName wx:val="Heading 9"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading9"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="8"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="character" w:default="on" w:styleId="DefaultParagraphFont">
<w:name w:val="Default Paragraph Font"/>
<w:semiHidden/>
</w:style>
<w:style w:type="table" w:default="on" w:styleId="TableNormal">
<w:name w:val="Normal Table"/>
<wx:uiName wx:val="Table Normal"/>
<w:semiHidden/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
<w:tblPr>
<w:tblInd w:w="0" w:type="dxa"/>
<w:tblCellMar>
<w:top w:w="0" w:type="dxa"/>
<w:left w:w="108" w:type="dxa"/>
<w:bottom w:w="0" w:type="dxa"/>
<w:right w:w="108" w:type="dxa"/>
</w:tblCellMar>
</w:tblPr>
</w:style>
<w:style w:type="list" w:default="on" w:styleId="NoList">
<w:name w:val="No List"/>
<w:semiHidden/>
</w:style>
<w:style w:type="paragraph" w:styleId="Title">
<w:name w:val="Title"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Title"/>
<w:spacing w:before="240" w:after="60"/>
<w:jc w:val="center"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="28"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="BodyText">
<w:name w:val="Body Text"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="BodyText"/>
<w:spacing w:after="120"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
</w:styles>
<w:body>
<wx:sect>
    <wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Title"/>
</w:pPr>
<w:r>
<w:t/>
</w:r>
</w:p>
<w:p>
<w:pPr>
<w:pStyle w:val="BodyText"/>
</w:pPr>
<w:r>
<w:t>
                    
                    
                </w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>objectifs journée</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>"business plan" associatif ?</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>modèle / activités responsabilités</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>articulations / LOG</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>SWOT</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t/>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>l'entreprise a aujourd'hui un potentiel important</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>compétences professionnel</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>citoyen</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>forte chance de réussite</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>apporter des idées et propsitions à des questions sociétales</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>notre manière d"y répondre avec notamment les technlogies</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>l'opportunité et la demande sont fortes aujourd'hui, avec peu de "concurrence"</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>ensemble de ressources "rares"</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>capacités de recherche et innovation</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>motivation du groupe et sens partagé entre membres</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>professionnellement : expérience collective et partage d'outils en pratique</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>ouverture vers mode de vie attractif perso / pro</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>potentiel humain, humaniste et citoyen</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>assemblage entre atelier et outillage</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>capacité de réponder en local et en global</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>associatif : contxte de crise multimorphologique / positionne référence en réflexion et usages</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>réseau régional et mondial de l'économie de la ,connaisance</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>asso prend pied dans le monde de la recherche</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>labo de l'innovation sociopolitique</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>acteur valable avec pouvoirs et acteurs en place</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>autonomie par prestations et services</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>triptique</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>éthique de la discussion</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>pari de la délégation</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>art de la décision</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>réussir à caler leprojet en adéquation avec le contexte actuel</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>assoc : grouper des personnes qui développent le concept</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>traduire les belles pensées au niveau du citoyen</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>compréhension</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>adhésion</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>ressources contributeurs réfréents</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>reconnaissance et référence exemplaires</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>financeements suffisants pour bien exister</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>notre organisation est claire</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>prendre des "marchés émergent"</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>double stratup avec succes-story</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>engageons une activité présentielle forte, conviviale et exemplaire</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>attirer de nouveaux membres locomotives</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>pratiquons en interne et externe une gouvernance explaire etune citoyennté de rêve</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Risques : cauchemars, dangers</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>disparition des forces vives, départ de membres actuels</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>opportunités atteignables mais difficile</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>difficultés de travailler ensemble dans la durée</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>risque de rater le train</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>sauter dans le dernier wagon et rester à la traîne</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>manquer de professionnalisme</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>perte de crédibilité</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>s'isoler entre nous et perdre le contact avec les autres acteurs</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>perdre la capacité de réponse au global</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>manque de concret, surdimension des reflexions</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>manque d'utilité socioplolitique</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>manque de nouveaux membres actifs, fidéliser</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>faire du surplace et</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>manque innovation</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t/>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>ne pas vivre ce que nous affirmons</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>cohérence entre langage gouvernance et la pratique</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>groupe de base insuffisant</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>non attractifs / nouveaux</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>pas ennuyants</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>pas efficaces en com</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>trop lent, rater l'opportunité actuelle</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>débordés par "concurrences"</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>départs de didier, micvhel, rené, corinne MCD etc</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>conclits de personnes et schisme entre 2 groupes ennemis</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>groupe amicale mais très merdique</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>système autocratique despotique ou sectaire</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t/>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
</wx:sect>
</w:body>
</w:wordDocument>
