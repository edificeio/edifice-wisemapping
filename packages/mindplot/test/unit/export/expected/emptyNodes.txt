1 
  [Note:  ]
	1.1 objectifs journée
		1.1.1 "business plan" associatif ?
		1.1.2 modèle / activités responsabilités
		1.1.3 articulations / LOG
	1.2 SWOT
		1.2.1 
			1.2.1.1 l'entreprise a aujourd'hui un potentiel important
				1.2.1.1.1 compétences professionnel
				1.2.1.1.2 citoyen
				1.2.1.1.3 forte chance de réussite
			1.2.1.2 apporter des idées et propsitions à des questions sociétales
			1.2.1.3 notre manière d"y répondre avec notamment les technlogies
			1.2.1.4 l'opportunité et la demande sont fortes aujourd'hui, avec peu de "concurrence"
			1.2.1.5 ensemble de ressources "rares"
			1.2.1.6 capacités de recherche et innovation
			1.2.1.7 motivation du groupe et sens partagé entre membres
			1.2.1.8 professionnellement : expérience collective et partage d'outils en pratique
			1.2.1.9 ouverture vers mode de vie attractif perso / pro
			1.2.1.10 potentiel humain, humaniste et citoyen
			1.2.1.11 assemblage entre atelier et outillage
			1.2.1.12 capacité de réponder en local et en global
			1.2.1.13 associatif : contxte de crise multimorphologique / positionne référence en réflexion et usages
			1.2.1.14 réseau régional et mondial de l'économie de la ,connaisance
			1.2.1.15 asso prend pied dans le monde de la recherche
			1.2.1.16 labo de l'innovation sociopolitique
			1.2.1.17 acteur valable avec pouvoirs et acteurs en place
			1.2.1.18 autonomie par prestations et services
			1.2.1.19 triptique
				1.2.1.19.1 éthique de la discussion
				1.2.1.19.2 pari de la délégation
				1.2.1.19.3 art de la décision
			1.2.1.20 réussir à caler leprojet en adéquation avec le contexte actuel
			1.2.1.21 assoc : grouper des personnes qui développent le concept
			1.2.1.22 traduire les belles pensées au niveau du citoyen
				1.2.1.22.1 compréhension
				1.2.1.22.2 adhésion
			1.2.1.23 ressources contributeurs réfréents
			1.2.1.24 reconnaissance et référence exemplaires
			1.2.1.25 financeements suffisants pour bien exister
			1.2.1.26 notre organisation est claire
			1.2.1.27 prendre des "marchés émergent"
			1.2.1.28 double stratup avec succes-story
			1.2.1.29 engageons une activité présentielle forte, conviviale et exemplaire
			1.2.1.30 attirer de nouveaux membres locomotives
			1.2.1.31 pratiquons en interne et externe une gouvernance explaire etune citoyennté de rêve
		1.2.2 Risques : cauchemars, dangers
			1.2.2.1 disparition des forces vives, départ de membres actuels
			1.2.2.2 opportunités atteignables mais difficile
			1.2.2.3 difficultés de travailler ensemble dans la durée
			1.2.2.4 risque de rater le train
			1.2.2.5 sauter dans le dernier wagon et rester à la traîne
			1.2.2.6 manquer de professionnalisme
				1.2.2.6.1 perte de crédibilité
			1.2.2.7 s'isoler entre nous et perdre le contact avec les autres acteurs
			1.2.2.8 perdre la capacité de réponse au global
			1.2.2.9 manque de concret, surdimension des reflexions
			1.2.2.10 manque d'utilité socioplolitique
			1.2.2.11 manque de nouveaux membres actifs, fidéliser
			1.2.2.12 faire du surplace et
				1.2.2.12.1 manque innovation
				1.2.2.12.2 
			1.2.2.13 ne pas vivre ce que nous affirmons
				1.2.2.13.1 cohérence entre  langage gouvernance et la pratique
			1.2.2.14 groupe de base insuffisant
			1.2.2.15 non attractifs / nouveaux
				1.2.2.15.1 pas ennuyants
			1.2.2.16 pas efficaces en com
			1.2.2.17 trop lent, rater l'opportunité actuelle
			1.2.2.18 débordés par "concurrences"
			1.2.2.19 départs de didier, micvhel, rené, corinne MCD etc
			1.2.2.20 conclits de personnes et schisme entre 2 groupes ennemis
			1.2.2.21 groupe amicale mais très merdique
			1.2.2.22 système autocratique despotique ou sectaire
			1.2.2.23 
